scour/scour.py

```diff
@@ -1022,6 +1022,15 @@ def removeDescriptiveElements(doc, options):
     return num
 
 
+def g_tag_is_unmergeable(node):
```

Comment:
Only very minor suggestion: Would you mind rewriting this as `g_tag_is_mergeable` instead? Avoids the double negation (`if not g_tag_is_unmergeable(child)`) below.